### DISCUSSION TITLE
Issue with opening attachaments

### DISCUSSION THREAD

**Discussion by piotrzawadz** [McShelby/hugo-theme-relearn#501] [Q&A] [answered]
I've followed documentation:
https://mcshelby.github.io/hugo-theme-relearn/shortcodes/attachments/index.html

1. Create folder structure with md file:
catalog1/catalog2/db_administration.md

2. Create folder:
catalog1/catalog2/db_administration.files

+ add several files into it

3. Then run hugo command to generate static files
 
4. Deploy them into github
 
Eventually files are properly displayed on the page generated from db_administration.md (styles work perfectly), however when I click attachment I receive:
"Woops. Looks like this page doesn't exist ¯\_(ツ)_/¯."

The generated link is:
https://<githubpages>/catalog1/catalog2/db_administration/.files/configure_db_txt

I've noticed that when it is change it manually to:
https://<githubpages>/catalog1/catalog2/db_administration.files/configure_db_txt

the file can be open. 

Therefore, it looks that / is somewhere added between db2_administration and .files. Is it a bug or should I change something in my code?

**Comment by McShelby** [comment]
I can not reporduce it. Do you have a public repository? What's the theme's version number?

**Comment by piotrzawadz** [comment]
it is the newest theme available. I've prepared public repository so you can easily check the behaviour:
https://piotrzawadz.github.io/hugo_tst/index.html

and then go to page:
https://piotrzawadz.github.io/hugo_tst/dbateam/databases/database/attachement/index.html

once you click any file you can see that issue is probably due to / added between attachement and .files: attachement/.files
example (original):
https://piotrzawadz.github.io/hugo_tst/dbateam/databases/database/attachement/.files/doc.txt

example (remove / and link works):
https://piotrzawadz.github.io/hugo_tst/dbateam/databases/database/attachement.files/doc.txt

**Reply by McShelby** [reply, marked as answer]
Thanks for submitting. This is a bug. See #503
